js/panel-manager.js

```diff
@@ -0,0 +1,57 @@
+// var count = 0;
+
+var PanelManager = {
+//$(document).ready(function(){
+//  $('#btn-add').click(function(e){
+  init: function(e){
+    $('.panel-manager').load('templates/panel-manager.html', function(e){
+      $('#slidebar').on('click', function(e){
+        $('#slider').toggle('slide', { direction: 'right' }, 300);
```

Comment:
Please use constants for the constant numbers, e.g. "PanelManager.TRANSITION_TIME". Looks like 300 is used repeatedly.